core/src/main/scala/org/apache/spark/ui/jobs/ExecutorTable.scala

```diff
@@ -92,16 +93,7 @@ private[ui] class ExecutorTable(stage: StageData, store: AppStatusStore) {
     executorSummary.toSeq.sortBy(_._1).map { case (k, v) =>
       val executor = store.asOption(store.executorSummary(k))
       <tr>
-        <td>
-          <div style="float: left">{k}</div>
-          <div style="float: right">
-          {
-            executor.map(_.executorLogs).getOrElse(Map.empty).map {
-              case (logName, logUrl) => <div><a href={logUrl}>{logName}</a></div>
-            }
-          }
-          </div>
-        </td>
+        <td>{if (k == "driver") k else k.toInt} </td>
```

Comment:
I'm curious, why does .toInt matter here? it ends up the same in the HTML, no?

Reply:
Thanks. I have updated.